comment_scan.rb

```diff
@@ -440,6 +440,36 @@ def restart(num_to_post, bundle)
         say reasons.join("\n")
       end
     end
+    command "!!/regexstats" do |bot, reason|
```

Comment:
I need to take a sec and loot at this block to see if it's pretty-able at some point soon.